src/core/net/ip6.cpp

```diff
@@ -210,8 +216,15 @@ ThreadError Ip6::SendDatagram(Message &message, MessageInfo &messageInfo, IpProt
     if (error == kThreadError_None)
     {
         message.SetInterfaceId(messageInfo.mInterfaceId);
-        mSendQueue.Enqueue(message);
-        mSendQueueTask.Post();
+
+        if (messageInfo.GetPeerAddr().IsRealmLocalMulticast())
```

Comment:
We should open an Issue to track it then. Maybe even an assert here for the time being that we aren't hitting larger than realm-local scopes.